### PR TITLE
Moved the frameit instructions to the bottom

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

File: docs/getting-started/ios/screenshots.md [hunk 1]
```diff
@@ -109,14 +146,16 @@ _frameit_ helps you beautify your screenshots with devices frames and text by ru
...
-```
+```no-highlight
 fastlane frameit
 ```
 
 This will only add a device frame around the screenshots, not the background and title. Those images can be used for your website, email newsletter and similar.
 
 If you want to implement the custom titles and background, you'll have to setup a `Framefile.json`, more information can be found [here](https://github.com/fastlane/fastlane/tree/master/frameit#titles-and-background-optional).
 
+If you want to upload the screenshots to the App Store, you **have** to provide a Framefile, with titles and background, otherwise the resolution of the framed screenshots doesn't match the requirements of iTunes Connect.
```
Comment (mfurtak) [inline docs/getting-started/ios/screenshots.md]: format `Framefile` ?

File: docs/getting-started/ios/screenshots.md [hunk 2]
```diff
@@ -94,6 +94,43 @@ fastlane snapshot                               # Generate screenshots for the s
...
+  deliver
+end
+```
+
+To get a list of all available options for each of the steps, run
+
+```no-highlight
+fastlane action snapshot
+fastlane action deliver
+```
```
Comment (mfurtak) [inline docs/getting-started/ios/screenshots.md]: 👍

**Comment by mfurtak** [comment]
Looks good except for one formatting suggestion. :ship: it when ready

File: docs/getting-started/ios/screenshots.md (hunk 1 above)
Comment (KrauseFx) [inline docs/getting-started/ios/screenshots.md]: YES 👍

**Pull request merged**